nixos/modules/misc/version.nix

```diff
@@ -8,8 +8,13 @@ let
     concatStringsSep mapAttrsToList toLower
     literalExpression mkRenamedOptionModule mkDefault mkOption trivial types;
 
+  # Attributes to exclude from escaping
```

Comment:
Please also explain *why* we are doing this in the comment.

Reply:
Ooof, forgot to remove it — it used to be above a list of `["ID" "ID_LIKE"]`